### PR TITLE
FFS-2976 Reflect invoicing changes in settlements API docs"

### PR DESCRIPTION
**Pull request by Thijs-Riezebeek**

### CODE REVIEW

File: source/reference/v2/settlements-api/get-settlement.rst [hunk 1]
```diff
@@ -222,12 +223,25 @@ Response
 
                    - The gross total costs for this payment method (includes VAT).
 
+          * - ``invoiceId``
+
+              .. type:: object
```
Comment (mollierobbert) [inline source/reference/v2/settlements-api/get-settlement.rst]:
```suggestion
              .. type:: string
```

File: source/changelog/v2/changelog.rst [hunk 2]
```diff
@@ -6,6 +6,17 @@ changes are documented here.
 March 2020
 ==========
 
+Monday, 23rd
+------------
+
+- It is now possible to link all costs in a settlement to all invoices associated with it.
```
Comment (shuisman) [inline source/changelog/v2/changelog.rst]: Maybe extend this a bit more with a reason

> Mollie is moving to a monthly invoicing system. As some settlements span multiple months, we have added an invoiceId to each period in a settlement.

File: source/reference/v2/settlements-api/get-settlement.rst [hunk 3]
```diff
@@ -222,12 +223,25 @@ Response
...
+
+            - The ID of the invoice that was created to invoice specifically the costs in this
+              month/period.
+
+              If an individual month/period has not been invoiced yet, then this field will not
+              be present until that invoice is created.
+
    * - ``invoiceId``
```
Comment (shuisman) [inline source/reference/v2/settlements-api/get-settlement.rst]: Should we make this deprecated to stimulate merchants / partners using the new invoiceId in the period object?

File: source/changelog/v2/changelog.rst (hunk 2 above)
Comment (mollierobbert) [reply source/changelog/v2/changelog.rst]: Great suggestion!

Comment (Thijs-Riezebeek) [reply source/changelog/v2/changelog.rst]: Incorporated the suggestion and reworded it a little bit to go with the rest.

File: source/reference/v2/settlements-api/get-settlement.rst (hunk 3 above)
Comment (Thijs-Riezebeek) [reply source/reference/v2/settlements-api/get-settlement.rst]: I think we should, but we'll have to wait until we actually can!

Comment (Thijs-Riezebeek) [reply source/reference/v2/settlements-api/get-settlement.rst]: So now that we can ,this is being picked up. Added a warning and mentioned the deprecation.

**Pull request merged**